src/renderer/i18n/nl.js

```diff
@@ -0,0 +1,483 @@
+    },
+    clearLayer: "Laag leegmaken...",
+    clearLayerQuestion: "Laag leegmaken?",
+    clearLayerPrompt: `Dit zal de laag terugzetten naar zijn standaard toestand.`,
+    copyFrom: "Kopieer van laag...",
+    pleaseSelectLayer: "Selecteer een laag...",
+    dualUse: "Modifier wanneer ingedrukt gehouden, anders een normale toets",
+    dualUseLayer: `Laag verandering wanneer ingedrukt gehouden, anders een normale toets`,
+    layoutMode: "Pas de toetsenbord layout aan",
+    colormapMode: "Pas de kleurmap aan",
+    importExport: "Importeer/Exporteer de huidige laag",
+    importExportDescription: `De data hieronder kan vrij aangepast worden of gekopieerd naar elders om terug te plakken voor ze ge├»mporteerd wordt. Dit is de interne representatie van de Chrysalis state, behandel met zorg.`,
+    loadDefault: "Laad een default:",
+    loadDefaultSuccess: "Default ingeladen!",
+    copyToClipboard: "Kopieer naar klipbord",
+    copySuccess: "Gekopieerd!",
+    pasteFromClipboard: "Plakken van klipbord",
+    pasteSuccess: "Geplakt!",
+    onlyCustom: {
+      warning: `Chrysalis ondersteunt niet langer configuraties die een mix van hardcoded and EEPROM lagen bevat. Als dit een feature is die je nodig hebt, dan horen we graag over jouw use case. In de meeste gevallen raden we echter aan om over te schakelen naar enkel aangepaste lagen, wat Chrysalis voor jou kan doen. Wanneer je de switch maakt zullen hardcoded lagen niet gebruikt worden, en de standaard laag die gezet wordt (als er een gezet wordt) is laag nul.`,
+      fixItButton: "Verander naar enkel aangepaste lagen",
+      openFR: "Open een feature request",
+    },
+    macros: {
+      title: "Macro #{{ index }}",
+      edit: "Macro aanpassen",
+      out_of_space: `Dynamische macros nemen te veel ruimte in, maak aub {{ overflow }} bytes vrij.`,
+      test: {
+        button: "Test",
+        placeholder: "Klik 'Test', en de macro zal hier afspelen.",
+      },
+      steps: {
+        time_ms: "{{ value }}ms",
+        in_ms: "ms",
+        unknown: "<onbekend>",
+        INTERVAL: "Vertraging tussen stappen",
+        WAIT: "Wacht",
+        KEYDOWN: "Hou toets ingedrukt",
+        KEYUP: "Laat toets los",
+        TAP: "Toets kort indrukken",
+        EXPLICIT_REPORT: "Zend geen rapport tussen stappen",
+        IMPLICIT_REPORT: "Zend een rapport tussen stappen",
+        SEND_REPORT: "Zend nu een rapport",
+        unsupported: "Niet ondersteund",
+      },
+    },
+  },
+  preferences: {
+    interface: "Gebruikersinterface",
+    advanced: "Geavanceerd",
+    verboseFocus: "Verbose logging",
+    devtools: {
+      main: {
+        label: "Ontwikkelaars tools",
+        description: `Deze voorkeuren helpen voornamelijk ontwikkelaars. Het kan zijn dat je gevraagd wordt om deze aan te zetten om meer informatie te kunnen verzamelen over een fout.`,
+      },
+      verboseLogging: {
+        label: "Verbose logging aanzetten",
```

Comment:
“Verbose logging” is a term mostly useful for programmers, perhaps it should indeed be kept English?

```suggestion
    verboseFocus: "Uitgebreid loggen",
    devtools: {
      main: {
        label: "Ontwikkelaars tools",
        description: `Deze voorkeuren helpen voornamelijk ontwikkelaars. Het kan zijn dat je gevraagd wordt om deze aan te zetten om meer informatie te kunnen verzamelen over een fout.`,
      },
      verboseLogging: {
        label: "Uitgebreid loggen aanzetten",
```

Reply:
Indeed, this will probably only be used by developers themselves.
Another case is where someone is asked to turn on verbose logging and if they have to do that, they're probably at a point where they could use a break and avoid having to search for the correct term in dutch.